cscs-checks/tools/profiling_and_debugging/gperftools_mpi_omp.py

```diff
@@ -72,14 +76,21 @@ def __init__(self, lang):
             # check job status:
             sn.assert_found('SUCCESS', self.stdout),
             # check txt report:
-            sn.assert_found('MPI_Allreduce', self.rpt_file_txt),
             sn.assert_found(
                 r'^\s+\d+ms\s+\d+.\d+%.*_jacobi.\w+:\d+', self.rpt_file_txt),
             # check pdf report:
             sn.assert_found('PDF document', self.rpt_file_doc),
         ])
+        self.perf_patterns = {
+            'jacobi_elapsed': self.report_flat_pctg,
+        }
+        self.reference = {
+            '*': {
+                'jacobi_elapsed': (0, None, None, '%'),
+            }
+        }
```

Comment:
You don't need the star anymore. ReFrame will automatically generate it, if it doesn't find it.

Reply:
can you add a comment in https://github.com/eth-cscs/reframe/blob/master/docs/running.rst#generating-a-performance-report ?

Reply:
It is mentioned in the tutorial: https://github.com/eth-cscs/reframe/pull/866/files